receiver/jaegerreceiver/trace_receiver_test.go

```diff
@@ -286,10 +286,9 @@ func expectedTraceData(t1, t2, t3 time.Time) ptrace.Traces {
 	return traces
 }
 
-func grpcFixture(t *testing.T, t1 time.Time, d1, d2 time.Duration) *api_v2.PostSpansRequest {
+func grpcFixture(t1 time.Time, d1, d2 time.Duration) *api_v2.PostSpansRequest {
 	traceID := model.TraceID{}
-	require.NoError(t, traceID.Unmarshal([]byte{0xF1, 0xF2, 0xF3, 0xF4, 0xF5, 0xF6, 0xF7, 0xF8, 0xF9, 0xFA, 0xFB, 0xFC, 0xFD, 0xFE, 0xFF, 0x80}))
-
+	traceID.Unmarshal([]byte{0xF1, 0xF2, 0xF3, 0xF4, 0xF5, 0xF6, 0xF7, 0xF8, 0xF9, 0xFA, 0xFB, 0xFC, 0xFD, 0xFE, 0xFF, 0x80}) // nolint:errcheck
```

Comment:
This PR seems to have reverted #14540, was it intentional? If not, would you mind to open a PR to bring back these changes @kovrus ?

Reply:
No, it was not an intentional change. I'll bring it back.

Reply:
Awesome, thanks :) Feel free to ping me on the PR if you want a review